assignments/B-02-81/app/models/user.js

```diff
@@ -0,0 +1,30 @@
+const mongoose = require('mongoose');
```

Comment:
You could have used more validation property . @sanyog96 has used in his PR just have look over it. Dont need to change it now but just remember it for future.